### DISCUSSION TITLE
How the packet reaches the WireGuard interface submariner？

### DISCUSSION THREAD

**Discussion by pengbinbin1** [submariner-io/submariner#2021] [General]
I have deployed my clusterset with 3 clusters use the cable driver wireguard. I found submariner will generate a  interface named `submarine` on the host and the interface type is wireguard. But i can not find any route for this interface,so how the packet reaches this interface ? allowed ips ? If so, what is the implementions about aloowed ips?

**Comment by sridhargaddam** [comment]
Submariner programs routes in `table 150` to redirect remote cluster traffic via the `submariner` (aka wireguard) interface. 
Following are the routes from a KIND cluster-2 Gateway node where `10.130.0.0/16`/`100.66.0.0/16` are remote cluster CIDRs.

```
bash-5.1# ip rule list
0:      from all lookup local
150:    from all lookup 150
32766:  from all lookup main
32767:  from all lookup default
bash-5.1# 
bash-5.1# ip route show table 150
10.130.0.0/16 dev submariner proto static scope link src 10.131.0.1 
100.66.0.0/16 dev submariner proto static scope link src 10.131.0.1 
bash-5.1# 
```

**Reply by pengbinbin1** [reply]
Thank you! @sridhargaddam
